sdks/python/apache_beam/ml/gcp/videointelligenceml_test_it.py

```diff
@@ -0,0 +1,75 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# pytype: skip-file
+
+"""An integration test that labels entities appearing in a video and checks
+if some expected entities were properly recognized."""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import unittest
+
+import hamcrest as hc
+from nose.plugins.attrib import attr
+
+import apache_beam as beam
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import matches_all
+
+# Protect against environments where Google Cloud VideoIntelligence client is
+# not available.
+try:
+  from apache_beam.ml.gcp.videointelligenceml import AnnotateVideoWithContext
+  from google.cloud.videointelligence import enums
+  from google.cloud.videointelligence import types
+except ImportError:
+  AnnotateVideoWithContext = None
+
+
+def extract_entities_descriptions(response):
+  for result in response.annotation_results:
+    for segment in result.segment_label_annotations:
+      yield segment.entity.description
+
+
+@attr('IT')
+@unittest.skipIf(
+    AnnotateVideoWithContext is None, 'GCP dependencies are not installed')
+class VideoIntelligenceMlTestIT(unittest.TestCase):
+  VIDEO_PATH = 'gs://cloudmleap/video/next/gbikes_dinosaur.mp4'
```

Comment:
Short pondering: Is it over-protective to have a separate bucket in e.g. Beam infra project in case Google would pull / update this file?

PR looks great btw. Just wanted to put that thought out there.

Reply:
Good point. I don't have enough permissions to create new files in Beam infra project, but I'll ask someone.
Until then I'll refrain from merging the PR.
Thanks!

Reply:
Thank you!

The same line of thinking would apply for the vision API sample too then.